pkg/core/resources/apis/mesh/virtual_outbound_helpers.go

```diff
@@ -0,0 +1,77 @@
+package mesh
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"text/template"
+
+	"github.com/asaskevich/govalidator"
+
+	mesh_proto "github.com/kumahq/kuma/api/mesh/v1alpha1"
+)
+
+func tagKeyOrName(parameter *mesh_proto.VirtualOutbound_Conf_TemplateParameter) string {
```

Comment:
If I understand it correctly we require to add a `parameter` entry even if a tag has the same key as a parameter in gotemplate?
So if we have template:
```yaml
host: {{ .foo }}.{{ .bar }}.{{ .baz }}
```
and we have DPP with inbound tags:
```yaml
tags:
  foo: value1
  bar: value2
  baz: value3
```
Anyway I have to provide parameters section:
```yaml
host: {{ .foo }}.{{ .bar }}.{{ .baz }}
parameters:
  - name: foo
  - name: bar
  - name: baz
```
Is it possible to resolve a template without explicit definitions of the parameters if no mapping required?